src/main/java/frc/team166/robot/Robot.java

```diff
@@ -132,8 +136,90 @@ public void testPeriodic() {
     }
 
     public Command CrossLineAndDropCube() {
-        return new CommandChain("Cross Line And Drop Cube").then(drive.DriveTime(3, 0.6), lift.RaiseLiftALittle())
-                .then(manipulator.CubeEject());
+        // return new CommandChain("Cross Line And Drop Cube").then(drive.DriveTime(1.8, 0.6), lift.LowerLiftForTime())
+        //       .then(lift.RaiseLiftALittle()).then(manipulator.CubeEject());
+        return new CommandChain("Cross Line And Drop Cube").then(lift.LowerLiftToLimitSwitch());
+    }
+
+    public Command MidAuto() {
+        String gameData;
+
+        gameData = DriverStation.getInstance().getGameSpecificMessage();
+        double degrees = 0.0;
+        if (gameData.length() > 0) {
+            if (gameData.charAt(0) == 'R') {
+                //"R" is for RIGHT NOT RED
+                degrees = 90.00;
+                //turning right                
+                System.out.println("Right");
+
+            } else {
+                degrees = -90.00;
+                //turning left               
+                System.out.println("Left");
+            }
+        }
+        Command cmdMidAuto = new CommandChain("Mid Auto").then(drive.DriveTime(.75, .6))
+                .then(drive.TurnByDegrees(degrees)).then(drive.DriveTime(.5, .6)).then(drive.TurnByDegrees(-degrees))
+                .then(drive.DriveTime(.3, .6), lift.RaiseLiftALittle());
+        return cmdMidAuto;
+
+    }
+
+    public Command ScaleAutoLeftStart() {
+        String gameData;
+        Command cmdLeftAuto = new CommandChain("Autonomous");
+        //The line above is to make VS Code happy and to take care of syntax errors
+        gameData = DriverStation.getInstance().getGameSpecificMessage();
+        double degrees = 0.0;
```

Comment:
I would just get rid of this variable and pass "90.0" into your command chains directly